mobify/progressive-web-scaffold
# Add to Homescreen Support

This PR adds add to homescreen support for Merlins on Chrome for Android. It also themes the address bar with the Merlin's brand colour.

Circle Lighthouse integration gives this branch 77/100, even though it doesn't seem to load the service worker correctly.

 **JIRA**: https://mobify.atlassian.net/browse/WEB-935
 **Linked PRs**: (links to corresponding PRs, optional)

## Changes
- Add a manifest file for Merlins
- Modify the loader to add the manifest to the page
- Remove manifest serving from the service worker (unnecessary since it can be hosted in the bundle)
- Add theme colour tag on load

## How to test-drive this PR
- Preview Merlins with this branch.
- In the Application tab of the dev tools, look for the Manifest panel
- Press the "add to homescreen" link in that panel
- See that the manifest information is populated. Without modifying the manifest to point to the local icon, it will fail until the icon is shipped in a production bundle
